src/app/(dev)/account/client-page.tsx

```diff
@@ -0,0 +1,219 @@
+"use client";
+
+import { useState } from "react";
+
+import { BackNavigationBar } from "@/components/BackNavigationBar";
+import { TextInput } from "@/components/TextInput";
+import { Button } from "@/components/ui/button";
+import type { PlayLevel } from "@/types/types";
+
+interface ClientAccountPageProps {
+  firstName: string;
+  lastName: string;
+  email: string;
+  playLevel: PlayLevel;
+  selectedLevel?: PlayLevel;
+  member: boolean;
+}
+// max-width = 450px
+const PLAY_LEVELS: PlayLevel[] = ["beginner", "intermediate", "advanced"];
+
+export default function ClientAccountPage({
+  firstName: initialFirstName,
+  lastName: initialLastName,
+  email: initialEmail,
+  playLevel: initialPlayLevel,
+  member,
+}: ClientAccountPageProps) {
+  const [firstName, setFirstName] = useState(initialFirstName);
+  const [lastName, setLastName] = useState(initialLastName);
+  const [email, setEmail] = useState(initialEmail);
+  const [playLevel, setPlayLevel] = useState<PlayLevel>(initialPlayLevel);
```

Comment:
could we perhaps use react-hook-form here?

not sure how it would interface with the play level selector though